lib/matplotlib/cbook/__init__.py

```diff
@@ -2242,7 +2242,7 @@ def type_name(tp):
                     type_name(type(v))))
 
 
-def _check_in_list(_values, **kwargs):
+def _check_in_list(_values, _print_supported_values=True, **kwargs):
```

Comment:
can you make this a keyword only argument?

Reply:
It's done. Thanks for the suggestion!